mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -36,11 +36,19 @@ def getConfigurations(fileName):
     configFile = open(fileName, 'r')
     lines = configFile.readlines()
     configs = [];
+    # All combinations of conv direction, type and layouts
```

Comment:
How about adding int8 to `DATA_TYPES` and then going with an
```python
if datatype == `convint8` and direction != `-F 1`:
    continue
```

It'll reduce code duplication

Reply:
(also, can you fix the extra space in the format string while you're here?)

Reply:
Nice catch! Thanks for catching those.